File: 卡码/KamaCoder/Python/test14.py
#14. 句子缩写
import sys

def main():
	n=int(input())
	for i in range(n):
		line=input().split()
		words=list(map(lambda s:s.capitalize(),line))
		for word in words:
			print(word[0],end="")
		print()

if __name__ == '__main__':
	main()

File: 卡码/KamaCoder/Js/test14.js
//14. 句子缩写
const rl=require("readline").createInterface({input:process.stdin});
const iter=rl[Symbol.asyncIterator]();
const readline=async ()=>(await iter.next()).value;

async function main(){
  const n=parseInt(await readline());
  for(let i=0;i<n;i++){
    let line=await readline();
    let words=line.split(/\s+/);
    words.forEach(item=>{process.stdout.write(item.toUpperCase().charAt(0))});
    console.log();
  }
}

main();